XpringKit/PayID/Generated/Models/Address.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Address.swift
+//
+// Generated by swagger-codegen
+// https://github.com/swagger-api/swagger-codegen
+//
+
+import Foundation
+
+
+
+public struct Address: Codable {
+
+    public var paymentNetwork: String
+    public var environment: String?
+    public var addressDetailsType: String
+    public var addressDetails: CryptoAddressDetails
+
+    public init(paymentNetwork: String, environment: String?, addressDetailsType: String, addressDetails: CryptoAddressDetails) {
+        self.paymentNetwork = paymentNetwork
+        self.environment = environment
+        self.addressDetailsType = addressDetailsType
+        self.addressDetails = addressDetails
+    }
+
+
+}
```

Comment:
I remember asking this before, but I forgot the answer. Why do we check in generated code again?

Reply:
also one for @keefertaylor

Reply:
Certain build systems in Swift (Carthage, Swift PM) pull in dependencies by checking out GitHub projects. These systems have hooks in them that run beforehand (and where we could generate this code) but the average user ended up having trouble with this because they didn't have the suite of tools installed they needed: grpc, node, git, protoc, and swagger. 

Instead, they'd get some random compile errors, think the project was broken because they didn't read the build logs and would open issues against the project. It's easier to just check them in than to force everyone who wants to use this project to install all the needed tooling.